sdks/python/apache_beam/runners/runner.py

```diff
@@ -187,7 +187,15 @@ def visit_transform(self, transform_node):
           logging.error('Error while visiting %s', transform_node.full_label)
           raise
 
+    # Replace the element type for inputs to a GroupByKey operation with a
```

Comment:
I would phrase this that some runners *require* a KV type for GBK inputs, so we enforce that here.